rubocop-airbnb/config/rubocop-layout.yml

```diff
@@ -248,10 +248,9 @@ Layout/IndentAssignment:
   Description: Checks the indentation of the first line of the right-hand-side of a
     multi-line assignment.
   Enabled: true
-  IndentationWidth:
 
 # Supports --auto-correct
-Layout/IndentHash:
+Layout/IndentFirstHashElement:
```

Comment:
The new name is `Layout/FirstHashElementIndentation`

Reply:
I think all of these `Indent` -> `Indentation` name changes are in rubocop 0.77, which isn't released yet.

https://github.com/rubocop-hq/rubocop/issues/7077